domain/storedetail/src/main/java/com/hankki/domain/storedetail/entity/StoreDetailResponseEntity.kt

```diff
@@ -1,15 +1,15 @@
-package com.hankki.feature.storedetail.model
+package com.hankki.domain.storedetail.entity
 
-data class MenuItem(
-    val name: String,
-    val price: Int
-)
-
-data class StoreDetail(
+data class StoreDetailResponseEntity(
     val name: String,
     val category: String,
     var isLiked: Boolean,
     val heartCount: Int,
     val imageUrls: List<String>,
     val menus: List<MenuItem>
 )
+
+data class MenuItem(
+    val name: String,
+    val price: Int
+)
```

Comment:
```suggestion
data class StoreDetailResponseEntity(

){
    data class MenuItem(
        val name: String,
        val price: Int
    )
}
```
만약 MenuItem을 단독으로 쓰지 않는다면 이렇게 묶어두는건 어떨까요~

Reply:
MenuItem을 단독으로 사용하고 있어서 따로 묶지 않도록 하겠습니다!